### ISSUE TITLE
keras model translation to pytorch doesn't perform as expected

### ISSUE THREAD

**Issue by emepetres** [abhishekkrthakur/approachingalmost#76]
Hi, as an exercise I implemented keras model in page 138 in pytorch.

It is working well, but I'm not able to pass from ~0.72, while with logistic regression I'm getting ~0.78. I believe that in keras the same model obtains ~0.83.

I've reviewed my code multiple times and I cannot find the difference with the keras one. In theory I'm using the same optimizer, but I had to change the loss to CrossEntropyLoss.

Has anyone any tip I should take into account translating it to pytorch? Is it possible that default values in pytorch are different from keras, and they are influencing the final results?

Thanks in advance, I leave my current code as reference:
* [Model](https://github.com/emepetres/cat-in-the-dat-ii/blob/main/src/models/embeddings.py)
* [Dispatcher](https://github.com/emepetres/cat-in-the-dat-ii/blob/main/src/model_dispatcher.py)
* [Train](https://github.com/emepetres/cat-in-the-dat-ii/blob/main/src/train.py)
